src/Nethermind/Nethermind.Blockchain/Tracing/GasEstimator.cs

```diff
@@ -125,18 +125,16 @@ private long BinarySearchEstimate(long leftBound, long rightBound, Transaction t
     }
 
     private bool TryExecutableTransaction(Transaction transaction, BlockHeader block, long gasLimit,
-        CancellationToken token)
+        CancellationToken token, EstimateGasTracer gasTracer)
     {
-        OutOfGasTracer tracer = new();
-
         Transaction txClone = new Transaction();
         transaction.CopyTo(txClone);
         txClone.GasLimit = gasLimit;
 
         _transactionProcessor.SetBlockExecutionContext(new(block, _specProvider.GetSpec(block)));
-        _transactionProcessor.CallAndRestore(txClone, tracer.WithCancellation(token));
+        TransactionResult result = _transactionProcessor.CallAndRestore(txClone, gasTracer.WithCancellation(token));
 
-        return !tracer.OutOfGas;
+        return result.Success && gasTracer.StatusCode == StatusCode.Success;
     }
 
     private class OutOfGasTracer : TxTracer
```

Comment:
What about this tracer? It seems like is no longer used (can't find any references to it and code coverage shows no usages).